programs/Makefile

```diff
@@ -131,11 +131,13 @@ mandir      ?= $(datarootdir)/man
 man1dir     ?= $(mandir)/man1
```

Comment:
A consequence of this patch is to make `man1dir` effectively inactive.
However, according to https://www.gnu.org/prep/standards/html_node/Directory-Variables.html : 

```
‘man1dir’
The directory for installing section 1 man pages. Write it as $(mandir)/man1.
```

Note that the variable is defined as _lowercase_.

So someone using : 
`make install man1dir=somePath`
won't be able to influence man installation directory.

Reply:
So, `MAN1DIR ?= $(MANDIR)/man1` should only be conditionally set.

Reply:
Looking at [1], automake doesn’t seem to allow to override it, if I am not mistaken.

[1]: http://www.rpi.edu/dept/acm/packages/fluxbox/fluxbox-0.1.7/doc/Makefile.in

Reply:
I don't see any `MANDIR` variable in the automake example.